source/config.go

```diff
@@ -74,6 +75,10 @@ type Config struct {
 	// LogreplAutoCleanup determines if the replication slot and publication should be
 	// removed when the connector is deleted.
 	LogreplAutoCleanup bool `json:"logrepl.autoCleanup" default:"true"`
+
+	// WithAvroSchema determines whether the connector should attach an avro schema on each
+	// record.
+	WithAvroSchema bool `json:"logrepl.withAvroSchema" default:"false"`
```

Comment:
Why logrepl only?

Reply:
Do we want to add this to the readme or is this for now just an "internal" feature we use for a special use case?

Reply:
> Why logrepl only?

because it is only enabled when in logrepl mode?

Reply:
I thought we might want to make this "more generic", but given that we're targeting a very specific use case, it's probably not relevant.:)